### PR TITLE
untested rename

### PR DESCRIPTION
**Pull request by ghost**
woo

### CODE REVIEW

**Comment by nate97** [comment]
Thank you for the help, however it's best to test these things before requesting for a pull. Thank you.

File: otp/avatar/DistributedPlayer.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def setWhisperSCFrom(self, fromId, msgIndex):
 
     def whisperSCCustomTo(self, msgIndex, sendToId, toPlayer):
         messenger.send('wakeup')
-        base.cr.ttiFriendsManager.d_whisperSCCustomTo(sendToId, msgIndex)
+        base.cr.triendsManager.d_whisperSCCustomTo(sendToId, msgIndex)
```
Comment (DarthMDev) [inline otp/avatar/DistributedPlayer.py]: typo

File: otp/login/LoginTTIAccount.py [hunk 2]
```diff
@@ -4,8 +4,8 @@
 import LoginBase
```
Comment (DarthMDev) [inline otp/login/LoginTTIAccount.py:4]: change the name of the file

**Comment by DarthMDev** [comment]
Yep this should work but u need to change ttifriendsmanager file to ttfriendsmanager
and also do stuff i commented on

**Comment by ghost** [comment]
This'll probably be finished up tomorrow astron was being a pain so I didn't manage to test my changes

**Comment by ghost** [comment]
due to way too many conflicts just gonna close this

**Pull request closed without merging**